src/picker.html

```diff
@@ -4,45 +4,162 @@
+    </div>
+  </div>
 
-  <table>
-    <tr>
-      <td>
-        <div id='simpleSearch'>
-          <select id='selectedTracker' class="miro-select miro-select--secondary-bordered">
-            <option value="" disabled selected hidden>Select Tracker ...</option>
-          </select>
-        </div>
-        <div id='advancedSearch'>
-          <input id='cbqlQuery' class="miro-input miro-input--primary miro-input--small"
-            placeholder="tracker.id in (12345)">
+  <div class="row">
+    <div>
+      <div id='simpleSearch'>
+        <label class="block-label">
+          Select Tracker
+        </label>
+        <select id='selectedTracker' class="miro-select miro-select--small miro-select--secondary-bordered">
+          <option value="" disabled selected hidden>Select Tracker...</option>
+        </select>
+      </div>
+      <div id='advancedSearch'>
+        <label class="block-label">
+          Enter CBQL query
           <a href="https://codebeamer.com/cb/wiki/871101" target="blank" title="Codebeamer cbQL documentation">
-            <img src="img/questionMark.svg">
+            <img id="questionMark" src="img/questionMark.svg">
           </a>
-        </div>
-      </td>
-      <td>
-        <button id='switchSearchButton' class="miro-btn miro-btn--primary miro-btn--small"></button>
-      </td>
-    </tr>
-  </table>
-  <table id='dataTable'>
+        </label>
+        <input id='cbqlQuery' class="miro-input miro-input--primary miro-input--small"
+        placeholder="e.g. tracker.id in (12345)">
+      </div>
+    </div>
+    <div id='switchButtonContainer'>
+      <button id='switchSearchButton' class="miro-btn miro-btn--primary miro-btn--small"></button>
+    </div>
+  </div>
+  <div id='table-container'>
+    <table id='dataTable'></table>
+  </div>
+  <footer>
+    <button id='importButton' class="miro-btn miro-btn--primary miro-btn--small">
+      <svg xmlns="http://www.w3.org/2000/svg" class="ionicon" viewBox="0 0 512 512"><path d="M336 176h40a40 40 0 0140 40v208a40 40 0 01-40 40H136a40 40 0 01-40-40V216a40 40 0 0140-40h40" fill="none" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="32"/><path fill="none" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="32" d="M176 272l80 80 80-80M256 48v288"/></svg>
+      <span id='importButtonText'></span>
+    </button>
```

Comment:
Would also be great to have a "import all button" instead of only for the displayed page